src/application/snapshot/widgets/organization-proposals.container.tsx

```diff
@@ -1,40 +1,25 @@
 import { useState } from 'react';
 
-import { useCommandQuery } from 'shared/messaging';
 import { Pagination } from 'shared/ui';
+import { ErrorBoundary } from 'shared/observability';
 
-import { GetProposalCommand } from '../commands';
+import { useProposalsQuery } from '../hooks';
 
 import { Proposal } from './proposal';
 
 interface Properties {
-  userHandle: string;
-  snapshotHandle: string;
   className?: string;
-  top?: number;
   onClose?: () => void;
+  officialName: string;
 }
 
-export const OrganizationProposalsContainer = ({
-  userHandle,
-  snapshotHandle,
-  className = 'fixed top-20',
-  top,
-  onClose,
-}: Properties) => {
+const Base = ({ className, officialName, onClose }: Properties) => {
   const [currentProposalIndex, setCurrentProposalIndex] = useState(0);
 
-  const proposalQuery = useCommandQuery({
-    command: new GetProposalCommand({
-      snapshotNames: [snapshotHandle],
-      pageNumber: currentProposalIndex,
-    }),
-    staleTime: Number.POSITIVE_INFINITY,
-    placeholderData: (previousData) => {
-      return previousData;
-    },
+  const proposalQuery = useProposalsQuery({
+    officialName,
+    pageNumber: currentProposalIndex,
   });
-
   const currentProposal = proposalQuery.data?.proposal;
 
   const isLoadingProposal =
```

Comment:
I think the isLoadingProposal can be changed to proposalQuery.isLoading, as we are not using placeholderData